content/en/Community/_index.md

```diff
@@ -25,9 +25,16 @@ This is a monthly meeting for the broader community where the Kyverno maintainer
 - Monthly on the second Thursday at 7:00 AM PST
 - [Repo](https://github.com/nirmata/office-hours-for-kyverno)
 
-### Contributors Meeting
+### Maintainers Meeting
 
-This is a weekly forum for Kyverno contributors and maintainers to discuss project delivery and implementation topics such as feature designs, fixes, code, and documentation:
+This is a public weekly meetings focused on topics relevant to plan and discuss upcoming releases, encouraging all contributors to actively participate and contribute to the discussions. It provides an opportunity for maintainers to come together with community members and collaborate on important topics related to the project. By fostering an inclusive environment, the Kyverno team aims to gather diverse perspectives, insights, and ideas that can further enhance the project's development and direction:
+
+- Weekly every Tuesday at 7:30 AM PST
+- [Agenda and meeting notes](https://docs.google.com/document/d/1I_GWsz32gLw8sQyuu_Wv0-WQrtRLjn9FuX2KGNkvUY4/edit?usp=sharing)
+
+### Community Meeting
+
+This is a weekly forum for Kyverno contributors and maintainers to discuss topics relevant to Kyverno development and other contributions. It’s meant for contributors and maintainers to meet, share, and get unblocked, peer review, and discuss all aspects of the Kyverno project:
```

Comment:
These descriptions sound too similar and too generic for me to understand. Maintainers Meeting is an internal but "open door" meeting where maintainers discuss issues and PRs pertaining to Kyverno's development and direction. The community are welcome to attend but may not raise agenda items. The Community Meeting is for Kyverno maintainers to make announcements and provide project updates, and this forum allows community members to raise agenda items of any sort, including but not limited to any PRs or issues on which they are working.

Reply:
I copied it from the meeting doc, how does it look now🤔